ingestors/media/tiff.py

```diff
@@ -23,7 +23,8 @@ def ingest(self, file_path, entity):
         entity.schema = model.get("Pages")
         pdf_path = self.make_work_file("tiff.pdf")
         self.exec_command(
-            "tiff2pdf", file_path, "-x", "300", "-y", "300", "-o", pdf_path
+            "tiff2pdf", file_path, "-n", "-j", "-x", "300", "-y", "300", "-o", pdf_path
```

Comment:
Could you explain why TIFF ingestion was previously failing with these to options enabled? It’s not obvious to me what they had an impact and why removing them fixes the issue.

Reply:
Also, did you figure out why only some TIFF files were affected and others not?

Reply:
@tillprochaska added a thorough explanation in the PR description, let me know if it is informative

Reply:
Thanks for the detailed explanation, this was really helpful.

One follow up question: My understanding of TIFF is very limited, but as TIFF can have multiple pages etc. and apparently embed image data using other image formats, is it possible that a TIFF file contains both JPEG compressed image data as well as other data?